5.6/apache/Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN set -xe \
 	&& gpg --batch --verify php.tar.xz.asc php.tar.xz \
 	&& rm -r "$GNUPGHOME"
 
-COPY docker-php-source /usr/local/bin/
+COPY docker-php-* /usr/local/bin/
```

Comment:
We only copy `source` here, since it is the only one required in the next `RUN`.  That way we don't have to rebuild php we only change one of the `ext` scripts.